cpp/tests/copying/slice_tests.cuh

```diff
@@ -219,8 +219,7 @@ inline std::unique_ptr<cudf::column> make_long_offsets_string_column()
   std::transform(iter, iter + num_chars, chars.begin(), [](cudf::size_type i) {
     return static_cast<int8_t>('a' + (i % 26));
   });
-  rmm::device_buffer d_chars(
-    num_chars, cudf::get_default_stream(), rmm::mr::get_current_device_resource());
+  rmm::device_buffer d_chars(num_chars, cudf::get_default_stream());
```

Comment:
Again, why not `cudf::get_current_device_resource_ref()`?

Reply:
Same answer above.